### ISSUE TITLE
Error in dynamic_raw_id.js

### ISSUE THREAD

**Issue by Letivan** [lincolnloop/django-dynamic-raw-id#107]
When clicking on the button with the dynamic_raw_id-clearfield class, nothing happens. Checked the dynamic_raw_id file.js there is an error in it - .closest('.v Foreign Key RowId AdminField, .vManyToManyRawIdAdminField'), does not find them because they are on the same level as the button
![2024-06-20_13-53-36](https://github.com/lincolnloop/django-dynamic-raw-id/assets/30967826/178a7cdd-ca89-42a8-a249-532cf47f7353)

I add .parent() and find(), redid it like this:
```
$('.dynamic_raw_id-clear-field').click(function() {
      const $this = $(this);
      $this
        .parent()
        .find('.vForeignKeyRawIdAdminField, .vManyToManyRawIdAdminField')
        .val('')
        .trigger('change');
      $this
        .parent()
        .find('.dynamic_raw_id_label')
        .html('&nbsp;');
    });
```

**Comment by shehantha08** [comment]
Check your version. in version 4.2 there is a bug, downgrade it to 4.0.

it works for me

**Comment by bartTC** [comment]
Thank you @Letivan! I applied your fix and did a v4.3 release.
